src/spesh/args.c

```diff
@@ -157,6 +157,12 @@ static void insert_getarg_and_box(MVMThreadContext *tc, MVMSpeshGraph *g, MVMSpe
                 hlltype_op = MVM_OP_hllboxtype_i;
                 fetch_op = MVM_OP_sp_getarg_i;
                 break;
+            case MVM_CALLSITE_ARG_UINT:
+                unboxed_temp = MVM_spesh_manipulate_get_temp_reg(tc, g, MVM_reg_int64);
```

Comment:
This doesn't need to be `MVM_reg_uint64`?

Reply:
You've got an eye for these things! Fixed it in the commit